.github/workflows/test_release_win32_nightly.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         $tmp=$tmp.psobject.properties.value.trim() 
         git clone https://github.com/awslabs/gluon-ts --branch $tmp
         cd gluon-ts
-        python -m pip install pip==20.2
+        python -m pip install -U pip
```

Comment:
This would just upgrade to latest, no? Does it makes sense to fix the pip version for the tests (as stuff *might* break if there is an update)?